sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -48,8 +48,8 @@ internal partial class TelemetryItem
             }
         }
 
-        public TelemetryItem(TelemetryItem telemetryItem, ActivitySpanId activitySpanId, ActivityKind kind, DateTimeOffset activityEventTimeStamp) :
-                        this("Exception", FormatUtcTimestamp(activityEventTimeStamp.DateTime))
+        public TelemetryItem(string name, TelemetryItem telemetryItem, ActivitySpanId activitySpanId, ActivityKind kind, DateTimeOffset activityEventTimeStamp) :
```

Comment:
Adding more parameters to constructor may cause issues at later point.

Reply:
@rajkumar-rangaraj what issues are you concerned about?
I think we could change some of these constructors to private

Reply:
This could be refactor later, its a minor suggestion and blocking the PR.